extract_unique_sensors_01.py

```diff
@@ -62,19 +62,19 @@ def extract_all_sensors(sensors_data, sensor_name):
     return candidate_lines
 
 # Loop over candidate_lines and extract sensor names
-def splitting_list(sensors_list):
+def splitting_list():
     """Splits the candidate_lines by dot and append that to sensors_list. It also removes the 
     empty lines.
     """
     sensors_list = []
     candidate_lines = sensors_list
-    sensors_list = extract_all_sensors(sensors_data)
-    unique_sensors = splitting_list(sensors_list)
+    sensors_list = extract_all_sensors(sensors_data = read_sensors(file_name, url), sensor_name=kat_sensor)
```

Comment:
What sorcery is this?

I thought you understood the last time we had a chat, that your function should never have access to any other function outside of it.
I remember telling you that, your functions should only know about the things inside it...
Not sure if that's clear or not.

This is something I expect:
```python
def splitting_list(sensor_list):
    """blablabla"""
    do something with sensor_list
    return sensor_list
```